src/components/PublicMessageBoard.module.css

```
@@ -0,0 +1,58 @@
+.public-message-board {
+  --message-board-border-color: #00000020;
+  --message-board-border-radius: 0.375rem;
+
+  border-collapse: separate;
+  border-spacing: 0;
+  width: 100%;
+}
+
+.public-message-board th {
+  padding: 1rem 0 1rem 1rem;
+  vertical-align: top;
+  width: 1px;
+}
+
+.public-message-board td {
+  padding: 1rem;
+}
+
+.public-message-board th,
+.public-message-board td {
+  border-top: 1px solid var(--message-board-border-color);
+}
```

```suggestion
.public-message-board th,
.public-message-board td {
  border-top: 1px solid var(--message-board-border-color);
}

.public-message-board tr:first-of-type > th,
.public-message-board tr:first-of-type > td{
  border-top: none;
}
```
Sorry, last thing, stops the weird double border which occurs at the top of the window